graphbolt/src/csc_sampling_graph.cc

```diff
@@ -472,5 +498,95 @@ torch::Tensor PickByEtype(
   return torch::cat(picked_neighbors, 0);
 }
 
+template <>
+torch::Tensor Pick<sampler_t::LABOR>(
+    int64_t offset, int64_t num_neighbors, int64_t fanout, bool replace,
+    const torch::TensorOptions& options,
+    const torch::optional<torch::Tensor>& probs_or_mask,
+    sampler_args<sampler_t::LABOR> args) {
+  if (fanout == 0) return torch::tensor({}, options);
+  if (probs_or_mask.has_value()) {
+    torch::Tensor picked_neighbors;
+    AT_DISPATCH_FLOATING_TYPES(
+        probs_or_mask.value().scalar_type(), "LaborPickFloatType", ([&] {
+          picked_neighbors = LaborPick<true, scalar_t>(
+              offset, num_neighbors, fanout, args.random_seed, options,
+              probs_or_mask, args.indices);
+        }));
+    return picked_neighbors;
+  } else {
+    return LaborPick<false>(
+        offset, num_neighbors, fanout, args.random_seed, options, probs_or_mask,
+        args.indices);
+  }
+}
+
+template <bool nonuniform, typename fp_t = float>
+inline torch::Tensor LaborPick(
```

Comment:
Can you add elaboration about how Labor Pick works? Similar to UniformPick / NonUniformPick above.